sqlmesh/core/scheduler.py

```diff
@@ -220,49 +220,53 @@ def run(
             restatements=restatements,
             ignore_cron=ignore_cron,
         )
-        dag = self._dag(batches)
+        if batches:
```

Comment:
I would suggest doing something like this instead:
```
if not batches:
   self.console.log_success("No models scheduled to run at this time.")
   return True
```
Then the rest remains unindented.